doc/sai-ptf/L3_test_plan.md

```diff
@@ -0,0 +1,620 @@
+5. Add a new route for ``DIP: IP:192.168.1.200/32`` with a next-hop on port2
+6. Add a neighbor with ``IP:192.168.1.200`` on Port1 with ``DMAC2``
+7. Send packet with ``DMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port5
+8. Received packet with the ``DMAC2``, ``SMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port1
+  
+- test_route_lpm_route_rif
+
+1. Make sure the route for ``DIP:192.168.1.0/24``is already configured (the route through next hop)
+2. Add a neighbor with ``IP:192.168.1.200`` on Port2 with ``DMAC1``
+3. Send packet with ``DMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port5
+4. Received packet with the ``DMAC1``, ``SMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port2
+5. Add a new route for ``DIP: IP:192.168.1.200/32`` and bind to port2 ``rif2`` directly
+6. Add a neighbor with ``IP:192.168.1.200`` on Port1 with ``DMAC2``
+7. Send packet with ``DMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port5
+8. Received packet with the ``DMAC2``, ``SMAC: SWITCH_MAC`` and ``DIP: IP:192.168.1.200`` on port1
+
+- test_svi_mac_flooding
+
+1. Check config for mac for port1~8 already exist
+2. Check already created route and neighbor as, route subnet ``192.168.1.0/24`` next hop to ``VLAN10`` SVI, neighbor for Port2-4 with IP ``DIP:192.168.1.92-94`` bind to ``VLAN10`` SVI
+3. Create neighbor for ``DIP:192.168.1.94`` with new ``DMAC:Mac4`` on port4 route interface 
+4. Send packet with ``DMAC: SWITCH_MAC``, ``SMAC:00:01:01:99:01:92`` and ``DIP: IP:192.168.1.94`` on port10
+5. No packet or flooding on VLAN10 member port
+
+- test_svi_mac_learning
+
+1. Run step 1-5 in test_svi_route_flooding
+2. No packet or flooding on VLAN10 member port(expect port2)
+3. For mac learning, send packet with ``SMAC:ServerX_MAC`` ``DMAC: SWITCH_MAC`` and ``DIP: Port10 Server_IP`` on port4 (learn ``Server_MAC`` on port4)
+4. Received packet with the ``DMAC:Port10 Server_MAC``, ``SMAC: SWITCH_MAC`` and ``DIP: Port10 Server_IP`` on port10
+5. For check mac learning from L2, send packet with ``DMAC: ServerX_MAC``, ``SMAC:Port2 Server_MAC``  on port2
+6. Received packet with the ``DMAC: ServerX_MAC``, ``SMAC:Port2 Server_MAC`` on port4
+
+- test_svi_mac_move
+
+1. Run step 1-6 in test_svi_route_learning
+2. Send packet with ``DMAC: SWITCH_MAC``, ``SMAC:ServerX_MAC`` and ``DIP: Port10 Server_IP`` on port3 (mac ``Server_MAC`` move to port3)
+3. Received packet with the ``DMAC:Port10 Server_MAC``, ``SMAC: SWITCH_MAC`` and ``DIP: Port10 Server_IP`` on port10
+4. For check mac learning from L2, send packet with ``DMAC: ServerX_MAC``, ``SMAC:Port2 Server_MAC``  on port2
+5. Received packet with the ``DMAC: ServerX_MAC``, ``SMAC:Port2 Server_MAC`` on port3
+
+- test_svi_mac_move_stress
+
+1. Run step 1-6 in test_svi_route_learning
+2. Send packet with ``DMAC: SWITCH_MAC``, ``SMAC:ServerX_MAC`` and ``DIP: Port10 Server_IP`` on port3 (mac ``Server_MAC`` move to port3)
```

Comment:
the SMAC:ServerX_MAC, should be learned ServerX_MAC to port3 other than Server_MAC?

Reply:
Yes